### ISSUE TITLE
Buttons options function as a list and should be indicated as such to screen reader users.

### ISSUE THREAD

**Issue by cryptasian** [microsoft/BotFramework-WebChat#4990]
### Is it an issue related to Adaptive Cards?

No.

### Are there any code-based customization done to Web Chat?

No, I am using Web Chat without any customizations except "styleOptions".

### What version of Web Chat are you using?

Latest production

### Which area does this issue affect?

Keyboard navigation

### What is the public URL for the website?

https://edfinancial.com

### How to reproduce the issue?

This is feedback from the accessibility company Allyant who has been testing our website and has people who use screenreaders and other assistive technologies as a part of their testing team.

![button-list](https://github.com/microsoft/BotFramework-WebChat/assets/30207740/9f46a974-60e3-4f46-aaf9-3cc3789b65ef)

### What do you expect?

When a series of similar items is grouped together in a meaningful or deliberate way, this should be conveyed to screen reader users. Use standard HTML to mark up all such content as a list.

### What actually happened?

The buttons are not conveyed as a list to screen reader users.

### WCAG reference

https://www.w3.org/WAI/WCAG22/Understanding/info-and-relationships

**Comment by compulim** [comment]
These buttons are grouped as toolbar, https://w3c.github.io/aria/#toolbar. They are grouped and implemented by following this APG pattern, https://www.w3.org/WAI/ARIA/apg/patterns/toolbar/.

Circled in the screenshot, the container has a role of toolbar and is grouping a collection of buttons.

![image](https://github.com/microsoft/BotFramework-WebChat/assets/1622400/a3b0656e-c11b-46ba-a86c-873a20a8abf5)
